controller/refugee_controller.py

```diff
@@ -1,42 +1,71 @@
 ###---- Manage Refugee Menu ----
 from datetime import date
-
+from enum import Enum
+from typing import Type
+from models.plan import Plan
 from models.camp import Camp
 from models.refugee import Refugee
+from controller.controller_error import ControllerError
+
+
+def list_medical_condition_types() -> Type[Enum]:
+    """
+    List all options for medical condition type.
+    """
+    return Refugee.MedicalCondition()
 
 
-def create_refugee(firstname: str,
+def create_refugee(user_id: str,
+                   firstname: str,
```

Comment:
Controller method signature should not be changed
user_id is not used anyway

Reply:
Thanks for the suggestion. The user_id has been removed.